strings/names/first.txt

```diff
@@ -957,7 +954,6 @@ Mariabella
 Mariah
 Mariana
 Marilene
-Mario
```

Comment:
There are 100.000 people in Spain alone (the source is INE) named Mario, to be honest I don't think it's necessary to delete this one!

Reply:
I know at least half a dozen people named mario, I think this is stupid to remove

Reply:
> Mario is more widely known than the spanish name

>Mario is the Italian, French, Croatian, Spanish, Portuguese, Bulgarian, Greek, and English form of the Latin Roman name Marius

Yeah you're right, it IS more widely known than just a spanish name. There's over a million people with this name.